crates/uv/tests/it/common/mod.rs

```diff
@@ -479,6 +479,7 @@ impl TestContext {
             .env(EnvVars::UV_TEST_PYTHON_PATH, self.python_path())
             .env(EnvVars::UV_EXCLUDE_NEWER, EXCLUDE_NEWER)
             .env_remove(EnvVars::UV_CACHE_DIR)
+            .env_remove(EnvVars::UV_TOOL_BIN_DIR)
```

Comment:
It looks like `setup-uv` sets this and we use `XDG_BIN_HOME` in tests which `UV_TOOL_BIN_DIR` takes precedence over.

Reply:
https://github.com/astral-sh/setup-uv/blob/3548439624127c489c1ca842eb263681394bc7b3/README.md?plain=1#L25-L26